conf/messages.en

```diff
@@ -340,4 +340,37 @@ notifications.reference = Reference
 notifications.action = Action
 notifications.dateOfResponse = Date of response
 notifications.roe = ROE
-notifications.soe = SOE
\ No newline at end of file
+notifications.soe = SOE
+
+# This part uses ActionType instances' values to differentiate between messages
+
+notifications.elem.title.Arrival = Arrive request
+notifications.elem.title.Departure = Depart request
+notifications.elem.title.DucrAssociation = Consolidation request
+notifications.elem.title.DucrDisassociation = Disassociate request
+notifications.elem.title.ShutMucr = Shut MUCR request
+
+notifications.elem.title.inventoryLinkingControlResponse = Request received
+notifications.elem.title.inventoryLinkingMovementResponse = Update message
+notifications.elem.title.inventoryLinkingMovementTotalsResponse = Update message
+notifications.elem.title.unspecified = Update message
+
+notifications.elem.timestampInfo.request = Request submitted {0} at {1}
```

Comment:
Form perspective of managin this - it would be easier to have single parameter here with correct formatting of time done on different level